automation_tools/__init__.py

```diff
@@ -3173,15 +3173,65 @@ def attach_subscription(url=None, consumer=None, pool_id=None, count=None):
         raise ValueError("Pass values under pool_id and count")
 
 
-def relink_manifest(manifest_file=None):
+def refresh_manifest(url, consumer, user, password):
+    """Task to remove all subscriptions from manifest and attach required ones.
+
+    :param url: Subscription Manager URL
+    :param consumer: A consumer hash to be used for getting the manifest
+    :param user: Red Hat Network username
+    :param password: Red Hat Network password
+    :returns: boolean True/False
+    """
+    auth_details = u'{0}:{1}'.format(user, password)
+
+    if isinstance(auth_details, str):  # py3
+        bytestring = bytes('{0}:{1}'.format(user, password), 'utf-8')
+    else:  # py2
+        bytestring = bytes('{0}:{1}'.format(user, password))
+
+    base64string = base64.encodestring(bytestring).strip()
+
+    command = ('curl -sk -X DELETE -H "Authorization:Basic {0}"'
+               ' "{1}/subscription/consumers/{2}/entitlements"').format(
+        base64string.decode('utf-8'), url, consumer)
```

Comment:
why not use python `requests` module for this?

Reply:
It was followed  from existing one :) : https://github.com/SatelliteQE/automation-tools/blob/master/automation_tools/__init__.py#L3167

Reply:
Yeah, i realized. I'm just thinking whether it wouldn't be better to use requests straight away

Reply:
I like the idea and would handle here https://github.com/SatelliteQE/automation-tools/issues/807 if that is fine with you.

Reply:
@rplevka ping :)